recipes/geosnap/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "geosnap" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fea40da099a1a8b01d9c5de97d35636eeb4a88770a740445f50cb0a2f6659ee2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - pandas
+    - giddy >=2.2.1
+    - libpysal
+    - spenc
+    - cenpy
+    - geopandas
+    - matplotlib
+    - scikit-learn
+    - seaborn
+    - pyarrow =0.13
+    - appdirs
+    - dash
+    - dash-bootstrap-components
+    - palettable
+    - pip
+    - tqdm
+    - quilt3
+    - xlrd
+    - region >=0.2.0
+    - tobler
+
+test:
+  imports:
+    - geosnap
+
+about:
+  home: http://github.com/spatialucr/geosnap
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Geospatial Neighborhood Analysis in Python'
+
+  description: |
+    geosnap is an open-source, Python package for exploring, modeling, and visualizing neighborhood 
+    dynamics. It provides a suite of tools for creating socio-spatial datasets, harmonizing those 
+    datasets into consistent set of time-static boundaries, and modeling neighborhood change using 
+    classic and spatial statistical methods.
+  doc_url: http://spatialucr.github.io/geosnap
+  dev_url: https://github.com/spatialucr/geosnap
+
+extra:
+  recipe-maintainers:
+    - knaaptime
+    - sjsrey
```

Comment:
@sjsrey - are you OK with being a maintainer?

Reply:
Yes on being a maintainer.